src/letter_grade.js

```diff
@@ -0,0 +1,65 @@
+function HTMLStrLetterGrade()
```

Comment:
This whole code should be in the Teacher.js file as it is used to generate webpages for the teacher. Once you have moved this code into the Teacher.js I will link your code to the server side so we can access the second webpage